examples/advanced/propagator/src/CMakeLists.txt

```diff
@@ -32,6 +35,11 @@ target_compile_definitions(${target} PUBLIC ${defs})
 target_include_directories(${target} PUBLIC
   $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>
   $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
+  $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/trackbase
+  $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/eventdisplay
+  $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/eventdisplay/tracks
+  $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/trackbase/
+  
```

Comment:
```suggestion
```

This is no longer needed (and allowed ;) ). Include directories are only set locally on the CMake library targets they belong to. By linking against such defined targets CMake will construct the proper include directory search path.